frontend/src/user/pages/ViewProfile/ViewProfile.tsx

```diff
@@ -75,35 +102,296 @@ export const ViewProfile = () => {
+              <tr>
+                <th>Členství</th>
+                <td>
+                  {formattedMemberships.length > 0
+                    ? formattedMemberships.map(memb => (
+                        <Fragment key={memb.category?.id}>
+                          <span className={styles.bold}>
+                            Organizační jednotka:{' '}
+                          </span>
+                          {memb.administration_unit}
+                          <br />
+                          <span className={styles.bold}>Typ členství: </span>
+                          {memb.category?.name}
+                          <br />
+                          <span className={styles.bold}>Členství v roce: </span>
+                          {memb.year}
+                        </Fragment>
+                      ))
+                    : '-'}
+                </td>
+              </tr>
+              <tr>
+                <th>EYCA nebo členská karta</th>
+                <td>
+                  {user.eyca_card?.valid_till ? (
+                    'ano, platnost do: ' +
+                    formatDateTime(user.eyca_card?.valid_till)
+                  ) : (
+                    <>
+                      ne <br />
+                      <ExternalButtonLink
+                        tertiary
+                        small
+                        target="_blank"
+                        href="https://www.eyca.cz/"
+                      >
+                        výhody EYCA evropské karty mládeže
+                      </ExternalButtonLink>
+                    </>
+                  )}
+                </td>
+              </tr>
+              <tr>
+                <th>Datum vzniku tvého profilu v BIS</th>
+                <td>
+                  {user.date_joined ? formatDateTime(user.date_joined) : '-'}
+                </td>
+              </tr>
             </tbody>
           </table>
-        </section> */}
+          <div className={styles.buttonGroup}>
+            <ButtonLink primary to="/user/akce/zucastnene">
+              Akce, kterých jsem se zúčastnil/a
+            </ButtonLink>
+            <ButtonLink primary to="/org/akce/vsechny">
+              Akce, které jsem organizoval/a
+            </ButtonLink>
+          </div>
+          <InfoBox>
+            <p>
+              Jako organizátor Hnutí Brontosaurus můžeš pro pořádání akcí
+              využívat podporu a zázemí HB. Zde najdeš užitečné informace a
+              pořádání akcí, které jsou dostupné pouze pod heslem
+            </p>
+            <ul>
+              <li>
+                <a
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  href="https://mozek.brontosaurus.cz/"
+                  className={styles.help}
+                >
+                  Mozek HB
+                </a>
+                , kde jsou všechny důležité dokumenty. Přihlašovací jméno:
+                brontosaurus, heslo: Brontosaurus40
```

Comment:
Seems really weird to add the password like this, not a good idea at all. But I suppose Hanka wanted that specifically.

Reply:
She did, but I will remain her to think about it during testing.